lib/cosmos/system/target.rb

```diff
@@ -136,7 +136,7 @@ def process_file(filename)
           parser.verify_num_parameters(1, 1, usage)
           begin
             require parameters[0]
```

Comment:
Should we break Cosmos.require_class into Cosmos.require_file and have require_class call require_file. I see duplicate code in interfaces/stream_interface.rb and io/tcpip_server.rb.